modules/apps/foundation/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/users.jsp

```diff
@@ -16,59 +16,61 @@
 
 <%@ include file="/facets/init.jsp" %>
 
-<c:if test="<%= !termCollectors.isEmpty() %>">
+<%
+com.liferay.portal.search.web.internal.facet.display.context.UserSearchFacetDisplayContext userSearchFacetDisplayContext =
+		new com.liferay.portal.search.web.internal.facet.display.context.UserSearchFacetDisplayContext(
+				facet,
+				fieldParam,
+				dataJSONObject.getInt("frequencyThreshold"),
+				dataJSONObject.getInt("maxTerms", 10),
+				dataJSONObject.getBoolean("showAssetCount", true));
+%>
```

Comment:
I got source formatted 😅 
Please collapse the lines above in a single line of JSP.
See: https://github.com/liferay/liferay-portal/commit/91f0d1233661785e5d22ee7875ef7b30e76f9e79